Dockerfile

```diff
@@ -6,6 +6,11 @@ ARG skip_frontend_build
 WORKDIR /frontend
 COPY package.json package-lock.json /frontend/
 COPY viz-lib /frontend/viz-lib
+
+# Controls whether to instrument code for coverage information
+ARG code_coverage
+ENV BABEL_ENV=${code_coverage:+test}
```

Comment:
@arikfr, I'll merge this PR as it feels safe + I can include it while updating #3991.

I'm highlighting the only change I had to do for the Dockerfile, in case you have any comments I can address in a follow up tomorrow.